server/command.go

```diff
@@ -174,6 +184,69 @@ func (p *Plugin) executeCommandQueue(args *model.CommandArgs) *model.CommandResp
 	return &model.CommandResponse{}
 }
 
+func calculateQueItemNumberAndUpdateOldItems(meeting *Meeting, args *model.CommandArgs, p *Plugin, hashtag string) (int, *model.CommandResponse) {
```

Comment:
It seems like this should go into a different file, because it doesn't inherently have to do with commands.

Reply:
done